letta/agent.py

```diff
@@ -896,7 +912,10 @@ def step(
         total_usage = UsageStatistics()
         step_count = 0
         while True:
-            kwargs["first_message"] = False
```

Comment:
I think this is referring to the *very* first message, so we shouldn't change this

Reply:
reverted and replaced indicator with `step=0`